### ISSUE TITLE
Who should be considered for getting access to the repo / considered for removing access from the repo

### ISSUE THREAD

**Issue by Hixie** [change-metrics/monocle#423]
It would be great if monocle could, for a given time range, list people who are members of the organization but have not actually contributed, and list people who are _not_ members of the organization and _have_ contributed. (Ideally this would include issues; see https://github.com/change-metrics/monocle/issues/421).

**Comment by TristanCacqueray** [comment]
I think we need to add members list to the project definition first, then we should be able to report inactive members.

That is a great suggestion, thank you @Hixie for all the feedback, it is much appreciated!

**Comment by morucci** [comment]
Yes the specification, at the moment, around users/groups, is to let the operator do the listing of users and user' groups in the configuration file. Synchronization of groups members will be up to the operator.

But that's true that It might be possible to have a DB index to store user groups and write a crawler to fill the database.

See this spec https://github.com/change-metrics/monocle/issues/307

So thanks for feedback! I think the feature your are requesting is something we should be able to tackle soon.

**Comment by TristanCacqueray** [comment]
The user group creation remains to be defined, but we made some progress for that enhancement with https://github.com/change-metrics/monocle/pull/438 . Though the group view only show review activity for the moment.
